Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -102,7 +102,26 @@
             %% 2. Parse modality
             
             pathLegacy_SubjectSession = fullfile(x.dir.xASLDerivatives, SubjectSession);
-            
+            bFolderExisted = exist(pathLegacy_SubjectSession, 'dir')==7;
+
+            % Backwards compatibility: rename sub-*** to sub-***_1
+            if strcmp(SessionID, '1')
+                pathLegacy_SubjectSession_OLD = fullfile(x.dir.xASLDerivatives, SubjectID);
+                if ~bFolderExisted && exist(pathLegacy_SubjectSession_OLD, 'dir')==7
+                    warning(['Renaming ' pathLegacy_SubjectSession_OLD '->' SubjectSession]);
+                    xASL_Move(pathLegacy_SubjectSession_OLD, pathLegacy_SubjectSession, 1, 0);
```

Comment:
A minor issue - we rename the folder, but don't rename the lock-files. So the old lock-files stay there. 

Shall we ignore that or keep that? Removing might be nice to avoid potential future issues. On the other hand, it's only relevant to old data. So just think about it and decide what you want ;)

Reply:
Good point, I will add this now by recursively checking the lock folder